docs/source/usage/workflows/synchrotronRequirements.py

```diff
@@ -0,0 +1,74 @@
+import numpy as np
+
+# Constants
+c = 299792458  # Speed of light in m/s
```

Comment:
could you use `scipy.constants` here instead?

Reply:
done